data/ships.txt

```diff
@@ -2136,6 +2176,60 @@ ship "Osprey"
 
 
 
+ship "Prospector"
```

Comment:
[Lol, why you gotta gype my name when you know you're doing it?](https://github.com/Amazinite/ES-Syndicate-Campaign/blob/master/images/ship/prospector.png)